doc/source/contributing_code.rst

```diff
@@ -261,6 +261,9 @@ Ensure that you have a working C/C++ compiler (e.g. gcc or clang). You will also
         cd ../testsuite/
         python -m pip install -e .
 
+
+.. note:: If installing  fails due to compiler ensure that you have the installed the build-essentials package. It can be installed with your OS/distro's package manager. 
```

Comment:
I think it might, and I just ended up installing build essentials to fix it

Reply:
Is build-essentials specific to linux? I can't remember how I get things going on mac, but I can usually `brew install gcc` etc (or it might come as a click-installable developer tools as part of xcode)

Reply:
It's specific to Debian flavours of Linux (so even more "obscure"). Unless we want to cover all major OS types and flavours I think the best that could be done is to just augment the C/C++ comment by saying you can get it from your favourite package manager?

Reply:
I agree with @IAlibay here -- @ALescoulie would you be ok making that change?